tutoroars/plugin.py

```diff
@@ -52,24 +64,24 @@
 # tutoroars/templates/oars/jobs/init/
 # and then add it to the MY_INIT_TASKS list. Each task is in the format:
 # ("<service>", ("<path>", "<to>", "<script>", "<template>"))
-MY_INIT_TASKS: list[tuple[str, tuple[str, ...]]] = [
-    # For example, to add LMS initialization steps, you could add the script template at:
-    # tutoroars/templates/oars/jobs/init/lms.sh
-    # And then add the line:
-    ### ("lms", ("oars", "jobs", "init", "lms.sh")),
+MY_INIT_TASKS: list[tuple[str, tuple[str, ...], int]] = [
+    ("oars", ("oars", "jobs", "init", "clickhouse-demo-xapi-data.sh"), 100),
```

Comment:
Is it right that they're all the same priority?

Reply:
Ah, good point... I'm not sure whether the order the tasks are added in applies another level of sorting on top of the priority ranking, but there's no reason to be ambiguous here.

Because `superset-add-admin.sh` must create the Admin user before `superset-api-dashboard.sh` can use it.. and really, `superset-init-security.sh` should set up the row-level permissions before `clickhouse-demo-xapi-data.sh` adds any xAPI data.

Reply:
Fixed with [7325b92](https://github.com/openedx/tutor-contrib-oars/pull/3/commits/7325b92b184d54ba6439831a3a32b4e861c02917) :)